### PR TITLE
OLS-644: Use version-aligned OCP docs for RAG - lightspeed-service

### PR DESCRIPTION
**Pull request by syedriko**
Run CI for lightspeed-service on OCP 4.15 and 4.16. Sister PR to https://github.com/openshift/release/pull/53210.

### CODE REVIEW

**Comment by syedriko** [comment]
@JoaoFula this is one of the PRs

File: ci-operator/config/openshift/lightspeed-service/openshift-lightspeed-service-main.yaml [hunk 1]
```diff
@@ -49,14 +49,20 @@ tests:
...
     mount_path: /tmp/secret
     name: ols-codecov-token
-- as: e2e-ols-cluster
+- as: e2e-ols-cluster-4-15
+  cluster_claim:
+    architecture: amd64
+    cloud: aws
+    labels:
+      region: us-east-2
+    owner: obs-logging
```
Comment (JoaoFula) [inline ci-operator/config/openshift/lightspeed-service/openshift-lightspeed-service-main.yaml:58]: owner is

`ols`

File: ci-operator/config/openshift/lightspeed-service/openshift-lightspeed-service-main.yaml [hunk 2]
```diff
@@ -117,7 +123,82 @@ tests:
...
           cpu: 100m
-    workflow: hypershift-hostedcluster-workflow
+    workflow: generic-claim
+- as: e2e-ols-cluster-4-16
+  cluster_claim:
+    architecture: amd64
+    cloud: aws
+    labels:
+      region: us-east-1
+    owner: obs-logging
```
Comment (JoaoFula) [inline ci-operator/config/openshift/lightspeed-service/openshift-lightspeed-service-main.yaml:133]: owner is

`ols`

File: ci-operator/config/openshift/lightspeed-service/openshift-lightspeed-service-main.yaml [hunk 3]
```diff
@@ -49,14 +49,20 @@ tests:
   secret:
     mount_path: /tmp/secret
     name: ols-codecov-token
-- as: e2e-ols-cluster
+- as: e2e-ols-cluster-4-15
+  cluster_claim:
+    architecture: amd64
+    cloud: aws
+    labels:
+      region: us-east-2
```
Comment (JoaoFula) [inline ci-operator/config/openshift/lightspeed-service/openshift-lightspeed-service-main.yaml:57]: @syedriko forgot about this. For 4.15, it's `us-east-1`

Comment (syedriko) [reply ci-operator/config/openshift/lightspeed-service/openshift-lightspeed-service-main.yaml]: @JoaoFula I don't think there's a 4.15 cluster pool like that:

```
   * could not run steps: step [release:latest-e2e-ols-cluster-4-15] failed: failed to find a cluster pool providing the cluster: map[architecture:amd64 cloud:aws owner:ols product:ocp region:us-east-1 version:4.15] 
```

If I search https://docs.ci.openshift.org/docs/how-tos/cluster-claim/ for `ols`, there's only one cluster pool with `ols` for owner:
```
openshift-observability-cluster-pool	ols-ocp-4-16-amd64-aws-us-east-1	0	0	1	3	ocp-release-4.16.1-x86-64-for-4.16.0-0-to-4.17.0-0	ols
```

**Comment by JoaoFula** [comment]
@syedriko thanks for spotting that. Just noticed I had made a typo in the creation of the 4.15 cluster pool.

**Comment by JoaoFula** [comment]
image
should be fixed now

**Comment by JoaoFula** [comment]
@syedriko seems like cluster-pools are functional. Test failures are unrelated to the cluster-claim.

**Comment by syedriko** [comment]
@xrajesh How do you feel about merging this and fixing the test failures as CI test failures in the OLS repo? Technically this branch does what it's supposed to - run the e2e tests against OCP 4.15 and 4.16.

**Comment by syedriko** [comment]
> @syedriko seems like cluster-pools are functional. Test failures are unrelated to the cluster-claim.

Thanks a ton, @JoaoFula !

**Comment by syedriko** [comment]
/pj-rehearse pull-ci-openshift-lightspeed-service-main-e2e-ols-cluster-4-15

**Comment by syedriko** [comment]
/pj-rehearse periodic-ci-openshift-lightspeed-service-main-e2e-ols-cluster-4-15

**Comment by syedriko** [comment]
/pj-rehearse pull-ci-openshift-lightspeed-service-main-e2e-ols-cluster-4-16

**Comment by syedriko** [comment]
/pj-rehearse periodic-ci-openshift-lightspeed-service-main-e2e-ols-cluster-4-16

**Comment by syedriko** [comment]
/pj-rehearse pull-ci-openshift-lightspeed-service-main-e2e-ols-cluster-4-16

**Pull request merged**